src/App.js

```diff
@@ -11,39 +13,263 @@ class App extends React.Component {
+          </p>
+        </>
+      );
+    }
+
+    const dealCardsButton = (
+      <>
+        <Button variant="contained" color="secondary" onClick={this.dealCards}>
+          Deal
+        </Button>
+      </>
+    );
+
+    const continueGameButton = (
+      <>
+        <Button
+          variant="contained"
+          color="success"
+          onClick={this.startNewMatch}
+        >
+          Yes, Enter Game
+        </Button>
+      </>
+    );
+
+    const exitGameButton = (
+      <>
+        <Button variant="contained" color="error" onClick={this.restartGame}>
+          No, Exit game
+        </Button>
+      </>
+    );
 
     return (
       <div className="App">
         <header className="App-header">
-          <h3>High Card 🚀</h3>
-          {currCardElems}
-          <br />
-          <button onClick={this.dealCards}>Deal</button>
+          <Box
+            sx={{
+              backgroundColor: "#4aedc4",
+              width: 700,
+              height: 500,
+              color: "black",
+            }}
+          >
+            <Typography style={{ fontFamily: "helvetica", fontSize: 20 }}>
+              <Box
+                sx={{
+                  width: 700,
+                  height: 90,
+                }}
+              >
+                <Box
+                  sx={{
+                    display: "flex",
+
+                    justifyContent: "space-evenly",
+
+                    alignItems: "center",
+                  }}
+                >
+                  <Box>
+                    <p>
+                      {this.state.numMatchesPlayer1Won} -- <b>Player 1</b>
+                    </p>
+                  </Box>
+
+                  <Box>
+                    <p>High Card 🚀</p>
+                    <p>
+                      {" "}
+                      Match: {this.state.matchNumber} --{" "}
+                      {this.state.cardDeck.length} Cards
+                    </p>
+                  </Box>
+
+                  <Box>
+                    {this.state.numMatchesPlayer2Won} -- <b>Player 2</b>
+                  </Box>
+                </Box>
+              </Box>
+              <div style={{ height: 50, width: 700 }}></div>
+              <Box
+                sx={{
+                  width: 700,
+                  height: 150,
+
+                  display: "flex",
+
+                  justifyContent: "space-around",
+                }}
+              >
+                {currCardElems}
+              </Box>
+
+              {this.state.hasGameBegin === true && (
+                <Box
+                  sx={{
+                    display: "flex",
+
+                    justifyContent: "space-evenly",
+                  }}
+                >
+                  <p>{this.state.player1Score}</p>
+                  <p>current scores</p>
+                  <p> {this.state.player2Score}</p>
+                </Box>
+              )}
+
+              {this.state.cardDeck.length > 0 && dealCardsButton}
```

Comment:
```suggestion
              {this.state.cardDeck.length && dealCardsButton}
```

Above 0 is truthy in JS